index.rst

```diff
@@ -10,7 +10,240 @@ This is complemented by the so called "run branch", which is a common branch use
+This is done by a common branch, centralized branch name that users can easily identify and contribute to.
+
+Here we descrive the procedures adopted by the run branch as well as how to manage their deployment.
+
+The Run Branch
+==============
+
+For every sprint (with a standard duration of 2 weeks), the observatory support team will create a ticket associated with the observatory operations support.
+The ticket number is used to define the associated run branch following the project guideline, i.e., ``tickets/<ticket-number>``.
+Branches are created in an as need base on the repositories that comprises the observing environment.
+It is not recommended to use the run branch for repositories outside the observing environment.
+However, in some cases (where hot patches are required), this might also be necessary.
+
+At the end of every sprint a member of the support team will curate the contributions from the run branche of each repository and either open pull requests to merge these contributions or work with the author of the contributions to get them merged.
+Nevertheless, not all contributions made to the run branch will be merged back during this process.
+Depending on what they are, the support team might request the author to create a dedicated ticket to host larger contributions.
+
+The following contributions are accepted as part of the run branch for any individual repository:
+
+- Critical hot fixes.
+  These are the most common usage of the run branch, and it is intended to allow patching things quickly at the production environment.
+- Minor updates to configuration.
+  This are most common on the Scheduler configuration in ``ts_config_ocs`` when an update to an existing survey configuration or block is needed.
+- Minor new features.
+  In general new feature should be avoided in the run branch.
+  However, minor new features are accepted in critical situations.
+  For example, a user may need a new configuration parameter on a SAL Script that exposes an existing functionality of the system that was initialy not exposed.
+
+Contributions that fall in the cathegory above can be pushed directly to the run branch and rolled out to the production environment.
+They may also be merged back at the close out of the run.
+
+Examples of contributions above are:
+
+- Small fix to a SAL Script or Observatory Control operation.
+- Small changes in an existing observing block.
+- Small changes in an existing Scheduler configuration.
+
+New functionality should not be directly pushed to the run branch.
+Instead, they should be developed as part of their own ticket and merged, at which point the author can coordinate with the support team to either rebase the run branch or cherry-pick the contributions.
+
+Examples of new functionality includes:
```

Comment:
Could be more explicit here to say "Examples of new functionality that should NOT be merged with the run branch includes: